demisto_sdk/commands/common/constants.py

```diff
@@ -394,6 +401,9 @@ class FileType(Enum):
 
 PACKS_LAYOUTS_CONTAINER_JSON_REGEX = fr'{PACKS_LAYOUTS_DIR_REGEX}\/layoutscontainer([^/]+)\.json'
 
+PACKS_PRE_PROCESS_RULES_DIR_REGEX = fr'{PACK_DIR_REGEX}\/{PRE_PROCESS_RULES_DIR}'
+PACKS_PRE_PROCESS_RULES_JSON_REGEX = fr'{PACKS_PRE_PROCESS_RULES_DIR_REGEX}/preprocessrule-([^/]+)\.json'
```

Comment:
we should allow the user to drop the prefix
```suggestion
PACKS_PRE_PROCESS_RULES_JSON_REGEX = fr'{PACKS_PRE_PROCESS_RULES_DIR_REGEX}/(?:preprocessrule-)?([^/]+)\.json'
```